client/platform/web-girder/views/ViewerLoader.vue

```diff
@@ -70,7 +70,7 @@ export default defineComponent({
         {{ dataset.name }}
       </span>
     </template>
-    <template #title-rigth>
+    <template #title-right>
       <export
         v-if="dataset !== null"
         :dataset-id="dataset._id"
```

Comment:
I think you're still missing the pipelines from here?

Reply:
Argh.   Right, sorry.

Reply:
Fixed in 84d2d7a

Includes another fix where the Viewer could load with stale imagery from the last dataset because I moved the data to a vuex module so it no longer gets wiped when the component dies.

Fixed with a simple loading flag.  This will have to change again in the electron branch, but I'm not sure how yet.